pkg/handler/processor/process/process.go

```diff
@@ -92,7 +94,7 @@ func processDocument(i *processor.Document) ([]*processor.Document, error) {
 }
 
 func preProcessDocument(i *processor.Document) error {
-	docType, format, err := guesser.GuessDocument(i)
+	docType, format, err := guesser.GuessDocument(logging.WithLogger(context.Background()), i)
```

Comment:
this should probably propagate context all the way to the top of what is calling this since the logger will be initialized by that.. using context like this doesn't quite match context usage patterns.

Reply:
you mean from `Process()` down to this function?

Reply:
yup!

Reply:
Done ✅